diagnostics/TC_MSE/doc/TC_MSE.rst

```diff
@@ -0,0 +1,132 @@
+desired. The user may comment out any of the plotting functions that are called in the Plotting.py script they do not want. 
+
+Required programming language and libraries
+-------------------------------------------
+
+Written using conda version 4.10.1 and python version 3.10
+
+matplotlib, numpy, pandas, xarray, scipy, sys
+
+Required model output variables
+-------------------------------
+
+The following 3-D (time-lat-lon) and 4-D (time-plev-lat-lon) variables are required:
+
+3-D Variables:
+
+Surface Downwelling Longwave Flux in Air, Units: W m-2, Frequency: 6-hourly 
+Surface Downwelling Shortwave Flux in Air, Units: W m-2, Frequency: 6-hourly
+Surface Upwelling Longwave Flux in Air, Units: W m-2, Frequency: 6-hourly
+Surface Upwelling Shortwave Flux in Air, Units: W m-2, Frequency: 6-hourly
+Surface Upward Latent Heat Flux, Units: W m-2, Frequency: 6-hourly
+Surface Upward Sensible Heat Flux, Units: W m-2, Frequency: 6-hourly
+Top of Atmosphere Outgoing Longwave Flux, Units: W m-2, Frequency: 6-hourly
+Top of Atmosphere Outgoing Shortwave Flux, Units: W m-2, Frequency: 6-hourly
+Top of Atmosphere Incoming Shortwave Flux, Units: W m-2, Frequency: 6-hourly
+
+4-D Variables:
+
+Air Temperature, Units: K, Frequency: 6-hourly instantaneous
+Geopotential Height, Units: m, Frequency: 6-hourly instantaneous
+Specific Humidity, Units: g g-1, Frequency: 6-hourly instantaneous
+
+In order to have sufficient samples, we recommend enough years of data to have at least 100 TCs, which is typically 2-5 years of simulation time depending on the resolution of the model.
+
+In the current implementation of the POD, pre-calcluated TC track data is also required as an input to the POD as obs data. If future versions incorporate TC tracking directly, additional model output variables will be required.
+
+References
+----------
+
+1. Wing, A. A., Camargo, S. J., Sobel, A. H., Kim, D., Moon, Y., Murakami, H., Reed, K. A., Vecchi, G. A., Wehner, M. F., 
+Zarzycki, C., & Zhao, M. (2019). Moist Static Energy Budget Analysis of Tropical Cyclone Intensification in High-Resolution Climate Models, 
+Journal of Climate, 32(18), 6071-6095, https://doi.org/10.1175/JCLI-D-18-0599.1.
+
+2. Dirkes, C. A., Wing, A. A., Camargo, S. J., Kim, D. (2022). Process-oriented diagnosis of tropical cyclones in reanalyses using a moist static
+energy variance budget, Journal of Climate (In review).
+
+More about this diagnostic
+--------------------------
+
+The MSE variance budget has been shown to capture the role of important physical processes in the development of TCs (Wing et al. 2016; Wing et al. 2019; Wing 2022). A budget for the spatial variance of column-integated MSE is given by:
+
+.. math::
+
+   \frac{1}{2}\frac{\partial \hat{h}'^2}{\partial t} = \hat{h}' F'_{k} + \hat{h}' N'_{L} + \hat{h}' N'_{S} - \hat{h}'(\widehat{\vec{u}\cdot \nabla  h})'.
```

Comment:
This is a great description of the POD. Thanks. Can you verify if the math symbols in the equations are resolved correctly?

Reply:
Hello Aparna, yes I can confirm all the math symbols in the equations are resolved correctly as I used the equation editor the framework docs provided at this link: https://editor.codecogs.com/